LSPS1/README.md

```diff
@@ -0,0 +1,386 @@
+# LSPS1 channel request
+
+| Name    	| base_api                      |
+|---------	|------------------------------	|
+| Version 	| 2                             |
+| OpenApi 	| [LSPS1.yaml](./LSPS1.yaml) 	|
+
+
+
+## Motivation
+
+The goal of this specification is to provide a standardized LSP API to purchase channels. Two client types are supported:
+
+**Wallets** that purchase a channel from the LSP directly.
+
+**Websites** that provide a UI for channel purchases. This should enable channel marketplaces and LSP channel purchase websites.
+
+To allow everybody to adopt this specification, we define a **HTTP** api. We purposefully decided against a lightning native api to not have to rely on any lightning implementation developers time and therefore reduced adoption potential.
+
+Some requirements are subtle; we have tried to highlight motivations and reasoning behind the results you see here. I'm sure we've fallen short; if you find any part confusing or wrong, please contact us and help us improve.
+
+## OpenAPI
+
+All specs are defined in the [OpenAPI](https://www.openapis.org/about) format. It can be viewed with various editors, most notably the [Swagger Editor](https://editor.swagger.io/) or the [vscode-openapi](https://marketplace.visualstudio.com/items?itemName=42Crunch.vscode-openapi) extension. All the OpenAPI documents include detailed descriptions of every field.
+
+## Definitions
+
+### Datetime
+
+All datetimes are represented as [ISO8601](https://en.wikipedia.org/wiki/ISO_8601) in the form of `2023-02-23T08:47:30.511Z` UTC.
+
+> **Rational** ISO8601 is human readable, it's a standard endorsed by [W3C](http://www.w3.org/TR/NOTE-datetime), and [RFC3339](https://www.rfc-editor.org/rfc/rfc3339), sortable, and supported widely.
+
+### Satoshi
+
+All satoshi values MUST be represented as string and NOT integer values. Make sure you check your integer limits.
+
+> **Rational** Plenty of json parsers use a 32bit signed integer for integer parsing. Max safe value is 2,147,483,647; 2,147,483,647sat = BTC21.474,836,47 which is too low.
+
+### Actors
+
+`LSP` is the API provider. `User` is the user of the API.
+
+### Channel sides
+
+Channel sides are seen from the user point of view. `local_balance` are the funds on the user side. `remote_balance` are the funds on the LSP side.
```

Comment:
Maybe better to be explicit and say `client_balance` and `lsp_balance` instead?

Reply:
Done.